osu.Game.Tests/Visual/Online/TestSceneShowMoreButton.cs

```diff
@@ -17,11 +18,11 @@ public class TestSceneShowMoreButton : OsuTestScene
 
         public TestSceneShowMoreButton()
```

Comment:
`ShowMoreButton` was renamed, but the test scene wasn't.

Reply:
It wasn't renamed. It's still there, but removed to another namespace

Reply:
OK, that's my bad, but in that case why does the test use `ProfileShowMoreButton` exclusively? Kind of confusing naming here.

Reply:
Will use TestButton or something